packages/ui/components/core/Footer.tsx

```diff
@@ -0,0 +1,91 @@
+import { Container, Title, Grid, Text, Stack, Group, createStyles } from '@mantine/core'
+
+import { Button } from './Button'
+import { SocialMediaIconButton } from './SocialMediaIconButton'
+import { BodyGrid } from '../layout'
+
+const useStyles = createStyles((theme) => ({
+	footer: {
+		backgroundColor: theme.other.colors.primary.footer,
+		['& a']: {
+			color: theme.colors.dark,
+		},
+	},
+}))
+
+const supportLinks = [
+	['Suggest an organization', '#'],
+	['Share feedback', '#'],
+	['Vetting process', '#'],
+	['Privacy statement', '#'],
+	['Anti-hate commitment', '#'],
+	['Digital accessibility', '#'],
+	['Disclaimer', '#'],
+]
+
+const connectLinks = [
+	['Inreach.org', '#'],
+	['Download the app', '#'],
+	['Subscribe to our newsletter', '#'],
+]
+
+const makeLinks = (links: string[][]) =>
+	links.map(([text, href]) => (
+		<Text key={text} component='a' href={href} fw={500}>
+			{text}
+		</Text>
+	))
+
+export const Footer = () => {
+	const { classes } = useStyles()
+
+	const support = makeLinks(supportLinks)
+
+	const connect = makeLinks(connectLinks)
+
+	return (
+		<Container className={classes.footer} fluid>
+			<BodyGrid style={{ padding: '40px' }} grow>
+				<Grid.Col md={5} sm={12}>
+					<Stack justify='space-between' style={{ height: '100%' }}>
+						<Stack align='start' spacing='xl'>
+							<Text>Inreach</Text>
+							<Title order={2} fw={500}>
+								{' '}
+								Seek LGBTQ+ resources. Reach safety. Find belonging
```

Comment:
All text based content need to be fed through the [i18next](https://react.i18next.com/) library - see other components for an example  ([MobileNav](https://github.com/weareinreach/InReach/blob/da5429b5afbb21eef87d19e91f443a03cd28759f/packages/ui/components/core/MobileNav.tsx#L68), [VerifiedBadge](https://github.com/weareinreach/InReach/blob/da5429b5afbb21eef87d19e91f443a03cd28759f/packages/ui/components/core/VerifiedBadge.tsx#L53))